README.md

```diff
@@ -72,6 +72,8 @@ Looking for inspiration? You can view the following profiles for an example:
 
 Your brand new file should look something similar to this one:
 
+> **Note** **CLI Profile Generator**: To generate the code for your profile using CLI, visit [Pradumna's Linkfree-CLI profile generator](https://github.com/Pradumnasaraf/LinkFree-CLI)
```

Comment:
I think @Pradumnasaraf's project is included in LinkFree, maybe we could also have steps to how to use it and link for further information.